cob_default_robot_config/robots/cob4-5/script_server/command_gui_buttons.yaml

```diff
@@ -69,7 +69,6 @@ group18:
     [front,move,front],
     [front_down,move,front_down],
     [back,move,back],
-    [back_down,move,back_down],
```

Comment:
we also should remove the configs from `cob_default_robot_config/robots/cob4-5/script_server/ 	head_joint_configurations.yaml`